config/settings/base.py

```diff
@@ -278,6 +281,11 @@
     ],
 )
 
+CELERY_WORKER_LOG_FORMAT = f"{LOG_FORMAT_START}, {LOG_FORMAT_END}"
+CELERY_WORKER_TASK_LOG_FORMAT = (
+    f"{LOG_FORMAT_START}, task_name=%(task_name)s, task_id=%(task_id)s, {LOG_FORMAT_END}"
+)
+
```

Comment:
For this Celery logging config, should we also log the processName here? Celery does this by default, but it looks like we won't anymore after this change.

Reply:
I don't find it useful. It usually says something like MainThread all the time.